google-cloud-storage/lib/google/cloud/storage/project.rb

```diff
@@ -471,6 +471,49 @@ def signed_url bucket, path, method: nil, expires: nil,
                             query: query
         end
 
+        ##
+        # Creates a new Storage::Bucket::Encryption instance.
+        #
+        # This method does not execute an API call. Use the returned encryption
+        # configuration when creating or updating a bucket with {#create_bucket}
+        # and {Bucket#encryption=}.
+        #
+        # @see https://cloud.google.com/kms/docs/ Cloud Key Management Service
+        #   Documentation
+        #
+        # @param [String] default_kms_key Resource name of the Cloud KMS
+        #   key, of the form
+        #   `projects/my-prj/locations/global/keyRings/my-kr/cryptoKeys/my-key`,
+        #   that will be used to encrypt files. The Service Account
+        #   associated with your project requires access to this encryption key.
+        #
+        # @return [Google::Cloud::Storage::Bucket::Encryption]
+        #
+        # @example Encrypt a new file with a default KMS key:
+        #   require "google/cloud/storage"
+        #
+        #   storage = Google::Cloud::Storage.new
+        #
+        #   kms_key_name = "projects/a/locations/b/keyRings/c/cryptoKeys/d"
+        #   encryption = storage.encryption default_kms_key: kms_key_name
+        #
+        #   bucket = storage.create_bucket "my-bucket" do |b|
+        #     b.encryption = encryption
+        #   end
+        #
+        #   bucket.create_file "path/to/local.file.ext",
+        #                      "destination/path/file.ext"
+        #
+        #   file = bucket.file "destination/path/file.ext"
+        #   file.kms_key #=> kms_key_name
+        #
+        def encryption default_kms_key: nil
+          encrypt_config = Storage::Bucket::Encryption.new
+          return encrypt_config if default_kms_key.nil?
```

Comment:
@quartzmo this is why `storage.encryption default_kms_key: nil` doesn't work, from my tests.

Reply:
@tswast Do you recall the purpose of this early return in Big Query's `Project#encryption`?

Reply:
Nope. Probably a bug in BigQuery, too honestly.